.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Motivation
+Add references to relevant tickets or a short description about what motivated you do it. (E.g Trello Card AND/OR GitHub ISSUE: https://github.com/integr8ly/tutorial-web-app/issues) 
+
+## What
+Add an short answer for: What was done in this PR? (E.g Don't allow users has access to the feature X.)
+
+## Why
+Add an short answer for: Why it was done? (E.g The feature X was deprecated.)
+
+## How
+Add an short answer for: How it was done? (E.g By removing this feature from ... OR By removing just the button but not its implementation ... ) 
+
+## Verification Steps
+Add detailed steps required to check this change. Following an example.
+ 
+1. Go to `XX >> YY >> SS`
+2. Create a new item `N` with the info `X`
+3. Try to edit this item 
+4. Check if in the left menu the feature X is not so long present.
+
+## Checklist:
+
```

Comment:
Wondering if we should have [ ] Screen shots included (if applicable)

Reply:
I really like the idea of reminding the individual creating the PR to include screenshots. I will add this now.